### ISSUE TITLE
Rebuilding a triggered build keep link to (previous) triggered build

### ISSUE THREAD

**Issue by tardyp** [buildbot/buildbot#3029]
This ticket is a migrated Trac ticket [3641](http://trac.buildbot.net/ticket/3641)

People contributed to the original ticket: `benoit.allard@...`, @tardyp
Ticket created on: `Nov 02 2016`
Ticket last modified on: `Feb 08 2017`

---

I have a builder that trigger some other builders.

When I rebuild one of the triggered build, the build is still linked as belonging to the parent build of the one I wanted to rebuild. I believe that link should be removed when rebuilding, and that no "Triggered from" should be showed.

---

**Comment by tardyp** [comment]
>  I believe that link should be removed when rebuilding, and that no "Triggered from" should be showed.

@benallard I disagree with this. I think it is useful to see what was the original parent.
in nine, a rebuild is really a 100% equal buildrequest. unlinking the parent would be breaking that.

**Comment by benallard** [comment]
Wouldn't it be better to have a "Rebuild of ..." instead of the "Triggered by ..." ?
